cloudsql-proxy/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gcr.io/cloudsql-docker/gce-proxy:1.19.1-alpine as builder
+
+FROM alpine:3.12.3
+
+LABEL source=git@github.com:kyma-project/kyma.git
+
+RUN apk add --no-cache \ 
```

Comment:
Please add an upgrade command as well to assure usage of latest package versions, see the other Dockerfiles in this repo as example